Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/MSBuildExecution/SimpleXmlLogger.cs

```diff
@@ -0,0 +1,117 @@
+﻿/*
+ * SonarScanner for MSBuild
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+using Microsoft.Build.Framework;
+
+// See MSDN for an example: https://msdn.microsoft.com/en-us/library/ms171471#Anchor_5
+// Example usage: msbuild x.csproj /logger:SonarScanner.MSBuild.Tasks.IntegrationTests.SimpleXmkLogger,..\SonarScanner.MSBuild.Tasks.IntegrationTests\bin\Debug\Logger.dll;log1.txt /noconsolelogger
+
+namespace SonarScanner.MSBuild.Tasks.IntegrationTests
+{
+    public class SimpleXmlLogger : ILogger
```

Comment:
_Dispose_ isn't called (possibly because the process is shutting down so the CLR doesn't bother). I've unregistered the events in the _Shutdown_ method.